### DISCUSSION TITLE
Attachments

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#7623] [Help] [answered]
Hi everyone,
I'm building an app where I need to manage files uploaded with the attachments tool. Does someone know where the file is stored after the front-end upload? 

Thanks

**Comment by melohagan** [comment, marked as answer]
MinIO! https://docs.budibase.com/docs/accessing-minio

**Reply by luigiPinnaAevo** [reply]
Thanks!!
